docs/docs/4-guides/3-generate-grid-form-views/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Generate Grid Form View
+---
+
+## Introduction
+
+This guide will walk you through creating a new DataGrid and Form in a Comet application.
+
+We will use the **API Generator** to create a new entity and the **Admin Generator** to generate the DataGrid for listing entities and the Form for editing them.
+
+Finally, we will integrate the generated DataGrid and Form into a new page and make it available from the Main Menu.
+
+![DataGridFormIntegration](./6-integrate-admin-generator-datagrid-into-comet-application/images/DataGridFormIntegration.gif)
+
+You can find the complete sample code for this guide [here](https://github.com/vivid-planet/comet-starter/compare/docs/api-admin-generator-test).
```

Comment:
It's likely that this branch will be deleted by accident in the future. Also, it will be outdated. Therefore we should either remove this sentence or create a separate example repo

Reply:
good idea - deleted.